tests/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ include(component/tests.cmake)
 include(connection/tests.cmake)
 include(coverage/tests.cmake)
 include(generator/tests.cmake)
+include(importer/tests.cmake)
 include(import_source/tests.cmake)
```

Comment:
Swap them: `import_source` should come before `importer`, if we are to respect alphabetical order.

Reply:
0ecc37d9